config/base_boxes.yaml

```diff
@@ -3,27 +3,47 @@ boxes:
     box_name:   'debian/jessie64'
     image_name: !ruby/regexp '/Debian.*Jessie/'
     pty:        true
+    scenarios:
+      - foreman
```

Comment:
Do we want the base boxes to have scenarios?

Reply:
I wasn't sure where to put it otherwise. Also note that this is just a variable that's later read by the BoxFactory to generate the scenario boxes based on it.

Reply:
Oh, hrmm, what is a "scenario" again in this context?

Reply:
Pretty much what the installer considers a scenario. I do derive that the content proxy scenario is available if the katello one is since we set the server property there.

Reply:
https://github.com/ekohl/forklift/blob/92b52547ece66a65786cfd0cf91549743fcc1cbc/lib/forklift/box_factory.rb#L55-L67 is the section that processes it.